 flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
As it builds from source, any reason to lock it to x86_64? aarch64 would be nice. Looks like it actually builds fine for it, but fails here: `cp: cannot stat 'fluent-reader/bin/linux/x64/linux*unpacked': No such file or directory`

Reply:
Let me remove that restriction :). I don't have aarch64 computer though.